index.html

```diff
@@ -91,30 +91,24 @@ <h3>Răzvan Cîrlugea</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/Toteanu-Mihai-Marius-poza.jpg" alt="Toteanu Mihai Marius" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Toteanu Marius</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    <span class="marius-quote">"At the end of the day, your great achievement is your smile"</span> starting from phrase that leads my life, I would describe myself as the person who never lose his optimism and shares it with all the people around, because, I believe that, even in the most challenging times, there is always a light, this bringing me to my favourite motto <span class="marius-quote">“The word impossible is not in my dictionary.”-Napoleon Bonaparte.</span>
```

Comment:
`At the end of the day, your great achievement is your smile` I think you wanted to say `At the end of the day, your greatest achievement is your smile`. Am I right?

Reply:
`starting from phrase that leads my life` should be `starting from this phrase that leads my life`

Reply:
`the person who never lose his optimism` should be `the person who never loses his optimism`

Reply:
`all the people around, because, I believe that` this phrase is way too long. Finish it and start a new one: `all the people around. I believe that`

Reply:
`dictionary.”-Napoleon` add some spaces `dictionary.” - Napoleon`